Makefile.inc

```diff
@@ -54,6 +54,15 @@ skipclippy:
 
 # Testing. A lot of directories can't test either, yet.
 test:
+	cargo test
 
 skiptest:
 	echo Not doing test for this target
+
+coverage:
+	cargo tarpaulin --out Lcov --output-dir "$(OREBOOT)/coverall"
+	cat `echo "$(OREBOOT)/coverall/lcov.info"`.info >> $(OREBOOT)/coverall/lcov.temps
+	rm "$(OREBOOT)/coverall/lcov.info"
```

Comment:
this is not necessary; for cleanup, we can include this in the `clean` target (can be done later also)